packages/@react-spectrum/button/src/index.ts

```diff
@@ -10,7 +10,7 @@
  * governing permissions and limitations under the License.
  */
 /// <reference types="css-module-types" />
-export {Button} from './Button';
+export {Button, pendingDelay} from './Button';
```

Comment:
😬

Reply:
haha, I'd be happy to just add a comment and use `1000` in both files if we wanna go that way instead, just felt like this was a innocuous const that we could use

Reply:
unfortunate that it's cross package...

Reply:
I'll go ahead and get rid of it then, should be fine with a comment